br/pkg/lightning/backend/local/duplicate.go

```diff
@@ -167,12 +277,12 @@ func (manager *DuplicateManager) sendRequestToTiKV(ctx context.Context,
 			}
 		}
 
-		if len(indexHandles) > 0 {
-			handles := manager.getValues(ctx, indexHandles)
-			if len(handles) > 0 {
+		if indexHandles.Len() > 0 {
```

Comment:
I have scan the new region list for this handle in `getValues`.

Reply:
The getValues logic is correct. I just mean the retry region logic at L269-271 seems not correct because the region key range may be already changed, though it's not belong to this PR.

Reply:
No. We will scan the regions again only when it get a region error